src/devices/tuya.ts

```diff
@@ -1396,6 +1396,77 @@ const definitions: Definition[] = [
             tuya.whitelabel('Moes', 'ZS-SR-EUD-3', 'Star ring smart dimmer switch 3 gangs', ['_TZE204_1v1dxkck']),
         ],
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_bxoo2swd']),
+        model: 'TS0601_dimmer_4',
+        vendor: 'TuYa',
+        description: '2 gang smart dimmer module',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            tuya.exposes.lightBrightnessWithMinMax().withEndpoint('l1'),
+            tuya.exposes.lightBrightnessWithMinMax().withEndpoint('l2'),
+            tuya.exposes.countdown().withEndpoint('l1'),
+            tuya.exposes.countdown().withEndpoint('l2'),
+            tuya.exposes.lightType().withEndpoint('l1'),
+            tuya.exposes.lightType().withEndpoint('l2'),
+            e.enum('power_on_behavior', ea.STATE_SET, ['off', 'on', 'previous']),
+        ],
+        meta: {
+            multiEndpoint: true,
+            tuyaDatapoints: [
```

Comment:
Looks very similar to https://github.com/Koenkk/zigbee-herdsman-converters/blob/f8e98d97fad60ce1085ccc869d33bb874767f551/src/devices/tuya.ts#L1312 , does e.g. `[21, 'backlight_mode', tuya.valueConverter.backlightMode],` also work for this device?

Reply:
All 4 switch / dimmer units in my PR are in-wall "brick" modules to connect to to an existing switch and lights. Backlight mode is not available. I guess the extra tuya DP (21) wouldn't cause issues, but the extra expose for backlight mode would confuse the end user. I haven't seen any devices where the behaviour (tuya datapoint and exposes) is "overwritten" in case of a specific sub-type of the device, this is why i added them as new devices.

Reply:
I see, then this is fine!